PaddleCV/PaddleVideo/configs/ets.yaml

```diff
@@ -0,0 +1,52 @@
+MODEL:
+  name: "ETS"
+  feat_size: 2048
+  fc_dim: 1024
+  gru_hidden_dim: 512
+  decoder_size: 512
+  word_emb_dim: 512
+  max_length: 80
+  beam_size: 3
+  START: "<s>"
+  END: "<e>"
+  UNK: "<unk>"
+  feat_path: '/paddle/Paddle-develop2/Paddle/PaddleProject/Paddle-caption/data/resnet152_features_activitynet_5fps_320x240.pkl'
```

Comment:
change this path to relative path, such as ./data/dataset/xxx

Reply:
Done